assets/images/Svg.js

```diff
@@ -0,0 +1,18 @@
+const React = require("react");
+const InlineSVG = require("svg-inline-react");
+
+const svg = {
```

Comment:
we already have an Svg.js here https://github.com/devtools-html/reps/blob/master/src/reps/images/Svg.js (which is similar).